extensions/fbc/src/org/sbml/jsbml/ext/fbc/converters/FbcV1ToCobraConverter.java

```diff
@@ -53,13 +55,129 @@
 
   Double defaultLowerBound = null;
   Double defaultUpperBound = null;
+  PackageDisabler packageDisabler;
+  
+  
+  private void createKineticLawForReaction(Reaction reaction) {
+    if (reaction == null) 
+    {
+      return;
+    }
+    reaction.unsetKineticLaw();
+    KineticLaw kineticLaw = reaction.getKineticLaw();
+    if (kineticLaw == null) 
+    {
+      kineticLaw = new KineticLaw(reaction);
+      LocalParameter fluxValueParameter = reaction.getKineticLaw().createLocalParameter(CobraConstants.FLUX_VALUE);
+      fluxValueParameter.setValue(0d);
+      fluxValueParameter.setUnits(CobraConstants.mmol_per_gDW_per_hr);
+      ASTNode astNode = new ASTNode(CobraConstants.FLUX_VALUE);
+      kineticLaw.setMath(astNode);
+    }
+    
+    LocalParameter upperBoundParameter = kineticLaw.getLocalParameter(CobraConstants.UPPER_BOUND);
+    if (upperBoundParameter == null) 
+    {
+      // 3. Assign lower and upper flux bounds if they were set, use defaults otherwise
+      upperBoundParameter = reaction.getKineticLaw().createLocalParameter(CobraConstants.UPPER_BOUND);
+      if (this.defaultUpperBound != null)
+      {
+        upperBoundParameter.setValue(this.defaultUpperBound);
+      } else 
+      {
+        upperBoundParameter.setValue(Double.POSITIVE_INFINITY);
+      }
+      upperBoundParameter.setExplicitlyConstant(true);
+      upperBoundParameter.setUnits(CobraConstants.mmol_per_gDW_per_hr);
+    }
+    
+    LocalParameter lowerBoundParameter = kineticLaw.getLocalParameter(CobraConstants.LOWER_BOUND);
+    if (lowerBoundParameter == null) 
+    {
+      lowerBoundParameter = kineticLaw.createLocalParameter(CobraConstants.LOWER_BOUND);
+      if (this.defaultLowerBound != null)
+      {
+        lowerBoundParameter.setValue(this.defaultLowerBound);  
+      } else 
+      {
+        lowerBoundParameter.setValue(Double.NEGATIVE_INFINITY);
+      }
+      lowerBoundParameter.setExplicitlyConstant(true);
+      lowerBoundParameter.setUnits(CobraConstants.mmol_per_gDW_per_hr);
+    }
+    
+    LocalParameter objectiveCoefficientParameter = kineticLaw.getLocalParameter(CobraConstants.OBJECTIVE_COEFFICIENT);
+    if (objectiveCoefficientParameter == null)
+    {
+      objectiveCoefficientParameter = kineticLaw.createLocalParameter(CobraConstants.OBJECTIVE_COEFFICIENT);
+      objectiveCoefficientParameter.setUnits(CobraConstants.mmol_per_gDW_per_hr);
+      objectiveCoefficientParameter.setValue(0d);
+    }
+  }
```

Comment:
Create kineticLaw and all four need local parameters for one reaction. Call this for each reaction in the model.
Objective coefficient has already been added here, which is being added with https://www.pivotaltracker.com/story/show/161591970